WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -37,7 +37,7 @@ class MeViewController: UITableViewController {
             NavigationItemRow.self,
             IndicatorNavigationItemRow.self,
             ButtonRow.self,
-            DestructiveButtonRow.self
+            DestructiveButtonRow.self,
```

Comment:
nit - did you intend to have this trailing comma for the last element in this Array?

Reply:
I did not, thanks for catching it!